### PR TITLE
Rimsha3

### PR DESCRIPTION
**Pull request by Rxmsha**

### CODE REVIEW

File: code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java
Comment (khushilad) [inline code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java:1]: did you delete the file?

Comment (Rxmsha) [reply code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java:1]: what file

Comment (Rxmsha) [reply code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java:1]: mainactivity? no

Comment (Rxmsha) [reply code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java:1]: oh wait org

Comment (Rxmsha) [reply code/app/src/main/java/com/example/the_tarlords/data/users/Organizer.java:1]: hold up lemme put it back

**Pull request closed without merging**